docs/src/meshes/p4est_mesh.md

```diff
@@ -12,3 +12,224 @@ such as the compressible Euler equations can use [`FluxRotated`](@ref) to
+**   1.986035130050921   0.054554577460044   0.0
+**   1.920916981799849   0.185149035378133   0.0
+**   1.768263070247418   0.329729152118310   0.0
+**   1.537500000000000   0.462500000000000   0.0
+**   1.295753434047077   0.606254343587194   0.0
+**   1.116583018200151   0.777350964621867   0.0
+**   1.024948365654583   0.934461926834452   0.0
+**   1.000000000000000   1.000000000000000   0.0
+**  7 2 4 1
+**  0 0 0 0
+```
+
+The last piece of information provided by `HOHQMesh` are labels for the different surfaces of an element.
+These labels are useful to set boundary conditions along physical surfaces.
+The labels can be short descriptive words.
+The label `---` indicates an internal surface where no boundary condition is required.
+
+It is important to note that these labels are given in the following order according to the
+local surface index `-x` `+x` `-y` `+y` as required by the [`p4est`](https://github.com/cburstedde/p4est) library.
+```
+**  Bezier --- Slant ---
+**  --- Right --- Top
+**  Bottom --- Right ---
+```
+
+For completeness, we provide the entire Abaqus mesh file for the example mesh in the figure above:
+```
+*Heading
+ File created by HOHQMesh
+*NODE
+1, 1.0, -1.0, 0.0
+2, 3.0,  0.0, 0.0
+3, 1.0,  1.0, 0.0
+4, 2.0,  0.0, 0.0
+5, 0.0,  0.0, 0.0
+6, 3.0,  1.0, 0.0
+7, 3.0, -1.0, 0.0
+*ELEMENT, type=CPS4, ELSET=Surface1
+1, 5, 1, 4, 3
+2, 4, 2, 6, 3
+3, 7, 2, 4, 1
+** ***** HOHQMesh boundary information ***** **
+** mesh polynomial degree = 8
+**  5 1 4 3
+**  0 0 1 1
+**   1.000000000000000   1.000000000000000   0.0
+**   1.024948365654583   0.934461926834452   0.0
+**   1.116583018200151   0.777350964621867   0.0
+**   1.295753434047077   0.606254343587194   0.0
+**   1.537500000000000   0.462500000000000   0.0
+**   1.768263070247418   0.329729152118310   0.0
+**   1.920916981799849   0.185149035378133   0.0
+**   1.986035130050921   0.054554577460044   0.0
+**   2.000000000000000                 0.0   0.0
+**                 0.0                 0.0   0.0
+**   0.035513826946206   0.105291711848750   0.0
+**   0.148591270347399   0.317731556850611   0.0
+**   0.340010713990041   0.452219430075470   0.0
+**   0.575000000000000   0.462500000000000   0.0
+**   0.788022294598950   0.483764065630034   0.0
+**   0.926408729652601   0.644768443149389   0.0
+**   0.986453164464803   0.883724792445746   0.0
+**   1.000000000000000   1.000000000000000   0.0
+**  4 2 6 3
+**  0 0 0 1
+**   2.000000000000000                 0.0   0.0
+**   1.986035130050921   0.054554577460044   0.0
+**   1.920916981799849   0.185149035378133   0.0
+**   1.768263070247418   0.329729152118310   0.0
+**   1.537500000000000   0.462500000000000   0.0
+**   1.295753434047077   0.606254343587194   0.0
+**   1.116583018200151   0.777350964621867   0.0
+**   1.024948365654583   0.934461926834452   0.0
+**   1.000000000000000   1.000000000000000   0.0
+**  7 2 4 1
+**  0 0 0 0
+**  Bezier --- Slant ---
+**  --- Right --- Top
+**  Bottom --- Right ---
+```
```

Comment:
In general I want to mention that this is again a great and thorough piece of documentation! Thanks for doing this, this is super helpful in the long term!!

Reply:
Thanks! It is sometimes annoying to document stuff, but I wanted a "clean" description of how the ABAQUS file works and what HOHQMesh adds.